crates/pallets/starknet/src/utils.rs

```diff
@@ -0,0 +1,64 @@
+pub fn get_project_path() -> String {
```

Comment:
Might make more sense to have the utils file in the node folder if possible

Reply:
It's not possible because of circular deps (same u got previously i think)